requirements.txt

```diff
@@ -12,4 +12,4 @@ django-solo==1.1.2
 Pillow==4.1.0
 django-modeltranslation==0.12.1
 django-embed-video==1.1.2
-openpyxl=2.4.8
+openpyxl==2.4.8
```

Comment:
<3

I caught it because I did a pip install trying to figure out the snugget case sensitivity issue- just good luck!